### DISCUSSION TITLE
Notepad++, add the vendors existing install options to the notepadplusplus package

### DISCUSSION THREAD

**Discussion by roysubs** [chocolatey-community/chocolatey-packages#2026] [Ideas]
Could you please add the following to this notepadplusplus package:

• Firstly, Notepad++ itself internally has a redirect flag to choose an install directory, but the Chocolatey package does not. You could simply utilise the existing /D flag in the Notepad++ install executable to allow the Chocolatey package to install to whatever folder a user wants. e.g. The installer syntax is `npp.8.4.5.Installer.exe /S /D="C:\Notepad++`   (/S is the silent flag).

• Secondly, you can auto-magically turn a "full install" Notepad++ into a "portable" Notepad++ simply by creating a single zero-length file named `doLocalConf.xml` inside the install folder, and this then will force all configuration files to be stored in the notepad++ folder instead of a users AppData folder. You could either add a separate portable package, or just add an option to the main Notepad++ chocolatey package to give users the ability to install the app in portable mode.

**Comment by TheCakeIsNaOH** [comment]
For the first point, you should be able to use the `--install-arguments` commandline option with `choco install` (and/or `upgrade) to pass that `/D="C:\Notepad++"` argument to the notepad++ installer.

For the second point, the portable version of notepad++ is already packaged:
https://community.chocolatey.org/packages/notepadplusplus.commandline
